.ci/proj_jjb.yaml

```diff
@@ -48,15 +48,15 @@
             description: "Run static analysis using cppcheck tool."
         - bool:
             name: "do_csbuild"
-            default: true
+            default: false
             description: "Run static analysis using csbuild tool."
         - bool:
             name: "do_style"
-            default: true
+            default: false
             description: "Analysis source code for coding style."
         - bool:
             name: "do_coverity"
-            default: true
+            default: false
```

Comment:
Why do you disable these checks by default? May be you forgot to revert debugging changes.